src/backend/base/langflow/api/v1/login.py

```diff
@@ -21,7 +21,7 @@
 
 
 @router.post("/login", response_model=Token)
-async def login_to_get_access_token(
+def login_to_get_access_token(
```

Comment:
`login_to_get_access_token` makes sync db calls and doesn't await so it's better to make it sync (fastapi will run it in a thread).

Reply:
LGTM